.circleci/scripts/setup_ci_environment.sh

```diff
@@ -32,7 +32,7 @@ if ! command -v aws >/dev/null; then
 fi
 
 if [ -n "${USE_CUDA_DOCKER_RUNTIME:-}" ]; then
-  DRIVER_FN="NVIDIA-Linux-x86_64-460.39.run"
```

Comment:
I had to update the Linux driver to the lates.
Otherwise docker build would fail with an error: docker: Error response from daemon: OCI runtime create failed: container_linux.go:380: starting container process caused: process_linux.go:545: container init caused: Running hook #0:: error running hook: exit status 1, stdout: , stderr: nvidia-container-cli: requirement error: unsatisfied condition: cuda>=11.5, please update your driver to a newer version, or use an earlier cuda container: unknown.
+ container_name=f713ab5eafbbd0f8f90448e83e8a09d9a1bda13923edbd86e10499c05475ebde
Error: Process completed with exit code 125.

Ref: https://github.com/pytorch/pytorch/runs/4327937766?check_suite_focus=true